content/docs/concepts/traffic-management/index.md

```diff
@@ -1053,43 +1000,16 @@ rules to:
   [multicluster Istio mesh](/docs/tasks/multicluster/gateways/#configure-the-example-services)
   on Kubernetes.
 
-You don’t need to add a service entry for every mesh-external service that you
+You don’t need to add a service entry for every external service that you
 want your mesh services to use. By default, Istio configures the Envoy proxies
-to passthrough requests from unknown services. You can also use service entries
-to configure internal infrastructure:
+to passthrough requests to unknown services, although you can't use Istio features
+to control the traffic to destinations that are not registered in the mesh.
 
-- A **mesh-internal** service entry adds a service running in the mesh, which
```

Comment:
There is confusion among users around mesh-internal and mesh-external service entries. The content here aimed to provide some clarity to alleviate said confusion. What changes can we make to the content for it to remain part of the topic?

Reply:
The confusion stems from the fact that we are trying to explain a difference when in fact every service entry is really mesh-external. Service entries are used to bring services into the mesh that are not otherwise there by the default registration adapter. Reading doc that says bringing a VM service, for example, into the mesh is somehow mesh-internal is the confusing part. The other things that we used to say is different (default TLS, for example) are not significant because that's all configurable, regardless of the location of the service, so we're just confusing people by trying to explain a difference, where fundamentally there is none.